src/PrimitiveRoots.jl

```diff
@@ -0,0 +1,63 @@
+using Random
+using Primes
+
+"""
+    _rand_root(𝔽, n)
+
+Obtain a random n-th root of unity (not necessarily primitive or ≢ 1)
+"""
+function _rand_root(rng::AbstractRNG, 𝔽::Type{<:PrimeField}, n)
+    # Assumes gcd(char(𝔽) - 1, n) == n
+    r = 𝔽(rand(rng, 1:(char(𝔽) - 1)))
```

Comment:
There's an implementation for `rand(F)` but for this purpose, it would need two improvements:
 - accept an AbstractRNG
 - have a way to specify nonzero
The former is just good practice and I've wanted the latter for unit testing.

If we find a way to represent this, all of this would work for `AbstractGaloisField`, I think?

Reply:
Yes, we could add a `MultiplicativeGroup{𝔽}` wrapper type such that you can have a method for `rand(rng::AbstractRNG, ::Type{MultiplicativeGroup{𝔽}})`.